.github/workflows/publish.yml

```diff
@@ -15,25 +15,12 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
       - name: Install dependencies
-        run: npm install
-      - name: Build puppeteer
-        run: npm run build
-      - name: Publish puppeteer
-        env:
-          NPM_TOKEN: ${{secrets.NPM_TOKEN_PUPPETEER}}
         run: |
-          npm config set registry 'https://wombat-dressing-room.appspot.com/'
-          npm config set '//wombat-dressing-room.appspot.com/:_authToken' '${NPM_TOKEN}'
-          echo "Publishing puppeteer"
-          npm publish
-          # DEPRECATED_RANGE=$(node utils/get_deprecated_version_range.js)
```

Comment:
Let's keep the code for the deprecation of old puppeteer releases (only Puppeteer though). There is a blocking issue in wombat that prevents it from working but we have to have this eventually.

Reply:
We can always look back at old releases to find the code. I don't see a purpose in keeping it here.